sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2146,7 +2146,9 @@ object SQLConf {
     .doc("When true, Spark tries to conform to the ANSI SQL specification: 1. Spark will " +
       "throw a runtime exception if an overflow occurs in any operation on integral/decimal " +
       "field. 2. Spark will forbid using the reserved keywords of ANSI SQL as identifiers in " +
-      "the SQL parser.")
+      "the SQL parser. 3. Spark will returns null for null input for function `size`. " +
+      "4. Spark will throw ArrayIndexOutOfBoundsException if invalid indices " +
```

Comment:
We can merge 1 and 4: `1. Spark will throw an exception at runtime if the inputs to a SQL operator/function are invalid, e.g. overflow in arithmetic operations and cast.`